INSTALL.md

```diff
@@ -342,6 +344,64 @@ ninja pycore
 ninja qgis_desktop
 ```
 
+### 3.7.1 Available compilation flags
+
+QGIS build is tunable according to your needs. Many flags are available to activate or deactivate some functionalities:
+
+* `WITH_3D`: Determines whether QGIS 3D library should be built
+* `WITH_ANALYSIS`: Determines whether QGIS analysis library should be built
+* `WITH_APIDOC`: Determines whether the QGIS API doxygen documentation should be built
+* `WITH_ASAN`: Use AddressSanitizer
+* `WITH_ASTYLE`: If you plan to contribute you should reindent with scripts/prepare_commit.sh (using 'our' astyle)
```

Comment:
This isn't really necessary anymore -- eg I don't build WITH_ASTYLE enabled. It was needed years ago when distros had older astyle which didn't support some of the code we used, but could likely be safely removed now.

Reply:
done